Resources/Maps/Shuttles/Mushroom.yml

```diff
@@ -0,0 +1,5827 @@
+      type: Transform
+  - uid: 382
+    components:
+    - pos: 0.5,-3.5
+      parent: 1
+      type: Transform
+  - uid: 383
+    components:
+    - pos: 0.5,-4.5
+      parent: 1
+      type: Transform
+- proto: CableTerminal
+  entities:
+  - uid: 384
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -2.5,-18.5
+      parent: 1
+      type: Transform
+  - uid: 385
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 0.5,-18.5
+      parent: 1
+      type: Transform
+- proto: CargoPallet
+  entities:
+  - uid: 386
+    components:
+    - pos: -5.5,-14.5
+      parent: 1
+      type: Transform
+  - uid: 387
+    components:
+    - pos: -7.5,-17.5
+      parent: 1
+      type: Transform
+  - uid: 388
+    components:
+    - pos: -7.5,-14.5
+      parent: 1
+      type: Transform
+  - uid: 389
+    components:
+    - pos: -5.5,-17.5
+      parent: 1
+      type: Transform
+- proto: ChairPilotSeat
+  entities:
+  - uid: 390
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -0.5,0.5
+      parent: 1
+      type: Transform
+  - uid: 391
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -1.5,0.5
+      parent: 1
+      type: Transform
+- proto: chem_master
+  entities:
+  - uid: 392
+    components:
+    - pos: 3.5,-7.5
+      parent: 1
+      type: Transform
+- proto: ClosetWallOrange
+  entities:
+  - uid: 393
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 5.5,-12.5
+      parent: 1
+      type: Transform
+    - air:
+        volume: 200
+        immutable: False
+        temperature: 293.1496
+        moles:
+        - 1.7459903
+        - 6.568249
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+      type: EntityStorage
+    - containers:
+        entity_storage: !type:Container
+          showEnts: False
+          occludes: True
+          ents:
+          - 395
+          - 394
+          - 397
+          - 396
+      type: ContainerContainer
+- proto: ClothingBeltMilitaryWebbing
+  entities:
+  - uid: 862
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingHandsGlovesCombat
+  entities:
+  - uid: 865
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingHeadHatHoshat
+  entities:
+  - uid: 858
+    components:
+    - desc: Yeehaw
+      name: Bounty Hunter Hat
+      type: MetaData
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+    - group: ""
+      type: NameIdentifier
+- proto: ClothingMaskGasMerc
+  entities:
+  - uid: 860
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingMaskMuzzle
+  entities:
+  - uid: 394
+    components:
+    - flags: InContainer
+      type: MetaData
+    - parent: 393
+      type: Transform
+    - canCollide: False
+      type: Physics
+    - type: InsideEntityStorage
+- proto: ClothingOuterCoatHoSTrench
```

Comment:
Hos gear?? this is sec gear.

Reply:
you can buy the sheriffs hat from an BountyVending and the trenchcoat is just renamed like it is on the sprinter

Reply:
Its getting fixed, the hat will need to get a new version with a new name..
Ill look into adding the 2 as new versions to be used for it, might as well.

Reply:
i believe its in now with the merc gear rework

Reply:
I have change the  custom bounty hunter name hos cloths To actual  Bounty hunter cloths